src/admin/components/SearchResult.js

```diff
@@ -26,14 +27,15 @@ const ResultLink = ({ children, end, middle, start, ...props }) =>
       'button-middle': middle,
       'button-start': start,
     })} rel="noopener noreferrer" target="_blank" {...props} >{children}</a>;
+
```

Comment:
I don't think it's convention. I had it right after since it felt like it should be in the function if it was possible. I would definitely put a space after the props if they came first so I guess the space does make sense.